src/box/lua/net_box.lua

```diff
@@ -759,6 +772,21 @@ function remote_methods:_request(method, opts, format, stream_id, ...)
         timeout = deadline and max(0, deadline - fiber_clock())
         self:wait_state('active', timeout)
     end
+
+    -- Ping is the only one, who need an error to be returned
+    -- So let's not change _request's interface only for that
```

Comment:
Firstly, 'commit' changes the interface, not the comment. Secondly, You don't need to do it like that. As I said, you should better stop throwing any errors from this function, not just for ping.

Reply:
It is used to decide whether error occured inside transport or not. Check out my comment above